x-pack/auditbeat/include/fields.go

```diff
@@ -11,7 +11,7 @@ import (
 )
 
 func init() {
-	if err := asset.SetFields("auditbeat", "fields.yml", Asset); err != nil {
+	if err := asset.SetFields("auditbeat", "fields.yml", 1, Asset); err != nil {
```

Comment:
I think this file is gone now from master. That’s probably when it’s showing `1`.